_data/accessibility-tests/banner.yml

```diff
@@ -0,0 +1,121 @@
+title: Banner
+component_status: pass
+test_items:
+# General tests
+  - summary: Color is not the only method used to convey meaning in the banner.
+    summary_additional: When viewing the banner, you never have to rely on color alone to understand meaning.
+    test_status: pass
```

Comment:
Two notes here

This is marked as conditional on the spreadsheet. I could see either being suitable for this test since the default is passing but it will be up to the user if they make any customizations.

But I'm also curious if this test is necessary for this component since it doesn't use color at all to convey meaning. The way I understand it is that if you use color to define something (i.e. Calling it "_green_ lock icon" instead of just "lock icon") it's not the only identifier. Since this doesn't really use any color to describe meaning, do we need this test?

Reply:
@amycole501 @alex-hull 
What do you think about @mahoneycm's comment?

Reply:
I think we could probably remove that check in the interest of streamlining the checks overall. I'm guessing not many people will adjust the colors or content to make the check necessary.

Reply:
I agree with Amy. But am also keeping in mind that some people (most likely not in the banner, and some people may even get rid of those icons in banner) will definitely use color in their icons to convey meaning. A red stop sign on screen could be easy for sighted people to understand, but without other context, would be difficult for those with visual disabilities.

Reply:
Update: After more discussion, the a11y team decided that this test should stay in because there is a chance users might make edits that could make this test necessary. Going to leave this test as-is. 

cc: @alex-hull and @amycole501